app/models/hackbot/interactions/sql.rb

```diff
@@ -0,0 +1,133 @@
+class Executer
+  def execute(query)
```

Comment:
@MaxWofford: Dropped the logic into #192.

I'm against making this only work as a slash command because it'd be helpful to share the results from SQL queries to public channels (ex. would have been awesome to do DB lookups inline in the convo we had at https://hackclub.slack.com/archives/C3P3J5N9J/p1492131511452004).

I think not publishing private data in channels should be left to the person executing the command because IMO there's no easy way around it.